client/src/main/java/io/pravega/client/stream/EventStreamWriter.java

```diff
@@ -50,13 +51,13 @@
      *        the same routingKey are guaranteed to be read in order. Two events with different routing keys
      *        may be read in parallel. 
      * @param event The event to be written to the stream (Null is disallowed)
-     * @return A future that will complete when the event has been durably stored on the configured number of
+     * @return A completableFuture that will complete when the event has been durably stored on the configured number of
```

Comment:
Minor: This line ending isn't aligned with the others, could you fix it, please?